src/utils.rs

```diff
@@ -4,18 +4,21 @@
 // Copyright (c) 2017 Guillaume Gomez
 //
 
+#[cfg(not(any(
```

Comment:
I'm wonderin about this `cfg`: should we instead list the supported systems? Because in here, the same problem will occur for openbsd for example...

Reply:
Yes. It makes sense. I'll try to find out how to improve it.

Reply:
Thanks!